PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+### Before you submit this, make sure you have:
+
+ - [ ] Ran format.py to ensure code is formatted
+ - [ ] Written tests if you added new features to WPILib
+ - [ ] Modified tests if you changed behavior
+ - [ ] Made corresponding changes in *both* Java and C++
+ - [ ] Made corresponding changes in simulation (again, Java and C++)
+
+### Things that will make it less likely for your PR (Pull Request) to be accepted:
```

Comment:
Here's a way to make this title less wordy: `Your Pull Request (PR) should avoid the following:`

In technical documents, an acronym should be introduced in parentheses after its first expanded usage.